src/Sylius/Bundle/ShippingBundle/Resources/translations/messages.pl.yml

```diff
@@ -29,6 +29,8 @@ sylius:
             description: Opis
             name: Nazwa
         shipping_method:
+            min_delivery_time_days: Minimalny czas dostawy (dni)
+            max_delivery_time_days: Maksymalny czas dostawy (dni)
```

Comment:
As above + additionally, at the end, it would be good to translate all added translations into available languages